pkgs/development/python-modules/scrapy/default.nix

```diff
@@ -82,6 +82,7 @@ buildPythonPackage rec {
   disabledTestPaths = [
     "tests/test_proxy_connect.py"
     "tests/test_utils_display.py"
+    #"tests/test_utils_python.py"
```

Comment:
If you comment them, you have to explain why you want to keep that comment. Alternatively, drop them!

Reply:
This was a leftover comment that's not needed anymore, will be removed.